04_pig_latin/pig_latin.rb

```diff
@@ -0,0 +1,39 @@
+def logic(str)
+  vowels = %w[a e i o u]
+
+  while !vowels.include?(str[0])
```

Comment:
In the Ruby world we don't really use `loop`, `while`, and others, we rather use `each`, `map` and so on (basically looping functions that are applied directly on the objects).

Can you think of a solution that uses `each` or `map`?

Reply:
I used `until` construct.